columnifier/parquet.go

```diff
@@ -103,11 +102,12 @@ func (c *parquetColumnifier) WriteFromFiles(paths []string) (int, error) {
 	var n int
 
 	for _, p := range paths {
-		data, err := ioutil.ReadFile(p)
+		f, err := os.Open(p)
```

Comment:
I guess this PR makes iterating processes reading some blocks of the file instead of reading the whole file. That's good.